### DISCUSSION TITLE
Report a user publishing copyrighted content that I do not own

### DISCUSSION THREAD

**Discussion by flyingcakes85** [community/community#21644] [New to GitHub]
<p>Hello all</p>
<p>I want to report a user who has been uploading copyrighted content on their repositories. I went to the report user page, where under the Copyright tab, there is the following text:</p>
<blockquote>
<p>If you believe that content on GitHub infringes a valid <strong>copyright you own</strong>, please see our <a href="https://docs.github.com/github/site-policy/dmca-takedown-policy" rel="noopener nofollow ugc">DMCA Takedown Policy</a> and our <a href="https://docs.github.com/github/site-policy/guide-to-submitting-a-dmca-takedown-notice" rel="noopener nofollow ugc">Guide to Submitting a DMCA Takedown Notice</a>.</p>
</blockquote>
<p>“Copyright you own” is the part that stopped me for now.</p>
<p>I am not a copyright holder for the content. I am not a legal spokesperson on behalf of the copyright owners. I am just a user who happened to stumble by the repository and found copyrighted content, and I want to bring it to GitHub’s notice so that they can take appropriate action.</p>
<p>I need to ask that how or where should I report such a violation, where I haven’t been harmed. I cannot use the “Report User” option since I am not the copyright holder.</p>
<p>I think this issue deserves attention, since the user is <em>effectively</em> using the GitHub infrastructure to share pirated content.</p>
<p><strong>What the user uploaded?</strong><br>
The user in question is maintaining several repositories which have educational content. Some of the repositories have e-books uploaded, and these ebooks are copyrighted by their owners. These are recent e-books and have a copyright notice at the beginning which the user is violating. I spent the evening looking through some of their repositories, and I have list of 11 ebooks they have uploaded and I want to report them. As I said, I am not the owner of these copyrights.</p>
<hr>
<p><em>I am not providing link to the user’s profile, since I don’t want to public shame them. If a GitHub staff needs info about the matter, they can send a message and I will share the info.</em></p>
<p>Thank You</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Only the copyright owner or their authorised representative can request the material be taken down, but you can of course report any copyrighted content to the owners. If they haven’t given permission for it to be shared then they will be able to file a takedown notice.</p>

**Reply by solohackerzorg** [reply]
hi
